site/en/guides/border-router/beaglebone-black.md

```diff
@@ -190,7 +161,11 @@ interface.
 interface. If you do not see the interface, check `journalctl` to see if the
 system is having difficulty bringing up the interface.
 
-## Step 3: Build and install OTBR
+Don't forget to pass the options `NETWORK_MANAGER=1` and
+`NETWORK_MANAGER_WIFI=1` to the setup script in the next step to use Network
+Manager.
+
```

Comment:
Does this reference `./script/setup`?  We probably need to show an example command either here or on the build page, otherwise it probably would easy to miss.

@sarah-iot thoughts?

Reply:
1. Agree @Vyrastas - I can add some examples on the build page as a follow-up to this PR; no problem. Quick q for @srickardti - i think that the bootstrap script checks for `dnsmasq` along with `network-manager`...for BBB is it required to pass:
`NETWORK_MANAGER=1 NETWORK_MANAGER_WIFI=1 ./script/bootstrap`

2. On this page, might want to move the note down to Step 4 just for clarity - maybe something like this?
See Build and Configuration for instructions on building and installing OTBR. To use Network Manager, make sure to pass the `NETWORK_MANAGER=1` and `NETWORK_MANAGER_WIFI=1` options for the [bootstrap]? and `setup` scripts.

Reply:
This was meant to ensure passing the options to `./script/setup`, but I realize that it would also be needed in the `./script/bootstrap` as well. That's where the packages are pulled from aptitude.